posttroll/subscriber.py

```diff
@@ -344,17 +339,19 @@ def _get_addr_loop(service, timeout):
         return self._subscriber
 
     def stop(self):
-        """Stop the subscriber.
-        """
+        """Stop the subscriber."""
         if self._subscriber is not None:
             self._subscriber.close()
             self._subscriber = None
 
 
-class Subscribe(NSSubscriber):
+class Subscribe(object):
```

Comment:
I don't think `object` is necessary here.

```suggestion
class Subscribe:
```

Reply:
How about this suggestion?

Reply:
Done. Also changed in other classes in the files touched by this PR.